app/models/proxy.rb

```diff
@@ -40,7 +40,7 @@ class Proxy < ApplicationRecord
 
   validates :oauth_login_url, format: { without: OAUTH_PARAMS }, length: { maximum: 255 }
 
-  validates :credentials_location, inclusion: { in: %w(headers query), allow_nil: false }
+  validates :credentials_location, inclusion: { in: %w(headers query authorization), allow_nil: false }
```

Comment:
The value `authorization` is important. That's what APIcast expects:
https://github.com/3scale/apicast/blob/288f06cf1822f74df250cdd99ce6056afa1e6f1c/gateway/src/apicast/configuration/service.lua#L116-L120
https://github.com/3scale/apicast/blob/288f06cf1822f74df250cdd99ce6056afa1e6f1c/gateway/src/apicast/configuration/service.lua#L81-L84